components/Footer.tsx

```diff
@@ -36,16 +38,20 @@ export default function Footer(): JSX.Element {
                 View Trip Planner
               </NavLink>
             </li>
-            <li>
-              <NavLink external href={process.env.API_DOCUMENTATION_URL}>
-                API documentation
-              </NavLink>
-            </li>
-            <li>
-              <NavLink external href={process.env.OTP_FORUM_URL}>
-                OTP Developers Forum
-              </NavLink>
-            </li>
+            {process.env.API_DOCUMENTATION_URL && (
```

Comment:
I think this should also include `API_MANAGER_ENABLED`, the documentation is not relevant otherwise. Probably the welcome screen should show the API docs link under these two conditions too.

Reply:
The condition is implicit as the component only renders if API_MANGER_ENABLED is true, and there is an fdot default. There are a few FDOT defaults...